addon/components/frost-button.js

```diff
@@ -69,11 +74,22 @@ export default Component.extend(PropTypeMixin, {
     text: PropTypes.string,
     title: PropTypes.string,
     type: PropTypes.string,
-    vertical: PropTypes.bool
+    vertical: PropTypes.bool,
+
+    // state
+
+    // keywords
+    attributeBindings: PropTypes.arrayOf(PropTypes.string),
```

Comment:
It is a bit annoying to have them duplicated though, maybe we can get `prop-types` to include them, or at least provide something we can add that let's us keep from the duplication, while still getting the type-checking.